ui-tests/cypress/support/pageObjects/content/types/EditPage.js

```diff
@@ -3,17 +3,17 @@ import {htmlElements} from '../../WebElement';
 import Content   from '../../app/Content.js';
 import KebabMenu from '../../app/KebabMenu';
 
-import AppPage      from '../../app/AppPage.js';
+import AdminPage      from '../../app/AdminPage.js';
 import DeleteDialog from '../../app/DeleteDialog';
 
 import TypesPage     from './TypesPage.js';
 import AttributePage from './attributes/AttributePage';
 
 export default class EditPage extends Content {
 
-  codeInput           = `${htmlElements.input}[name=code]`;
-  nameInput           = `${htmlElements.input}[name=name]`;
-  saveButton          = `${htmlElements.button}.AddContentTypeFormBody__save--btn`;
+  nameInput  = `${htmlElements.input}[name="entityTypeDescription"]`;
```

Comment:
This is not related to the creation of the _AdminPage_, please move them to a separate commit